tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py

```diff
@@ -370,14 +370,15 @@
                     item="37",
                     name='EDUCATION_LEVEL',
                     friendly_name="education level",
-                    type='number',
+                    type='string',
                     startIndex=55,
                     endIndex=57,
                     required=False,
                     validators=[
                         validators.or_validators(
                             validators.isInLimits(0, 16), validators.isInLimits(98, 99)
```

Comment:
I believe these should now be updated to `isInStringLimits`. Also, if we don't want to match `00` we should remove that value from limits of the other validators.